template-skeleton/template/src/components/views/login/UserMenu.js

```diff
@@ -34,12 +33,12 @@ function UserMenu() {
     return null;
   }
 
-  // User is NOT logged in, so display Login with CARTO
+  // User is NOT logged in, so display nothing
   if (!user) {
-    return <UserMenuLogin />;
+    return null;
```

Comment:
`return;` is fine?

Reply:
I don't have a preference. Is there any official recommendation on that?

Reply:
I saw that we return `null` in several components, for me we can leave it like this :+1: